README.md

```diff
@@ -153,7 +153,7 @@ Copyright (c) 2015, Jad Bitar and licensed under [The MIT License][mit].
 [milestones]://github.com/gourmet/platform/issues/milestones
 [migrations/repo]://github.com/cakephp/migrations
 [mit]:http://www.opensource.org/licenses/mit-license.php
-[mobiledetect/repo]://github.com/mobiledetectlib/mobiledetectlib
+[mobiledetect/repo]://github.com/serbanghita/Mobile-Detect
```

Comment:
Visit http://github.com/mobiledetectlib/mobiledetectlib in your browser and you will see :wink: